packages/pyodide-kernel-extension/schema/kernel.v0.schema.json

```diff
@@ -25,6 +25,21 @@
       "default": [],
       "format": "uri"
     },
+    "pipliteInstallDefaultOptions": {
+      "type": "object",
+      "description": "Default options to pass to piplite.install",
+      "default": {},
+      "properties": {
+        "indexUrls": {
```

Comment:
I would reckon this should be `index_urls`, and then wherever possible, the parent object handed around as a blob of JSON-compatible kwargs.